neutral_generation/constants.py

```diff
@@ -61,7 +61,8 @@
     'was': 'were',
     'has': 'have',
     'does': 'do',
-    'goes': 'go'
+    'goes': 'go',
+    'quizzes': 'quiz'  # 1-1-1 doubling rule
```

Comment:
I agree, I think listing out all the words is not really a great practical solution. I put in the "quizzes" --> "quiz" mapping since I couldn't really think of other verbs that also drop a consonant when they become plural in the third-person. I'll see if I can gather other examples for this doubling rule.